app/editor/src/features/content/tool-bar/constants/ActionHelper.ts

```diff
@@ -0,0 +1,7 @@
+import { ContentTypeName, IActionModel } from 'hooks';
+
+export type ActionHelper = {
+  init?: boolean;
+  determineActions: () => (a: IActionModel) => boolean;
```

Comment:
I find the `determineActions` property to be hard to understand.

Reply:
Actually an unused file 😛 deleted!